job_templates/snippet/publisher_cobertura.xml.em

```diff
@@ -8,6 +8,9 @@
       <zoomCoverageChart>false</zoomCoverageChart>
       <maxNumberOfBuilds>0</maxNumberOfBuilds>
       <failNoReports>false</failNoReports>
+      <lineCoverageTargets>80, 0, 0</lineCoverageTargets>
+      <methodCoverageTargets>80, 0, 0</methodCoverageTargets>
+      <conditionalCoverageTargets>70, 0, 0</conditionalCoverageTargets>
```

Comment:
I assume these are the default values suggested by the plugin? Can you describe the semantic meaning of these values: first value and two zeros.

Reply:
7bc11f7 adds comments describing the values.

First: coverage score above which is considered 100% healthy
Second: coverage score below which is considered 0% healthy
Third: coverage score below which is considered an unstable build.

The current values are the defaults.

Reply:
Please mention that these are the default value in the comment in the template.

Reply:
I thought about doing so but it seems like an easy comment to forget to delete if we do decide to changes those values.

Reply:
Added one in de99381 anyway because I don't feel very strongly about it.